source/_integrations/miele.markdown

```diff
@@ -0,0 +1,155 @@
+  - **Temperature**: Represents the current temperature in refrigerators, freezers, and ovens. Entities are created for up to 3 zones depending on the device capabilities.
+{% enddetails %}
+
+## Automation examples
+
+Get started with these automation examples
+
+### Send a notification when the appliance ends the program
+
+{% details "Example YAML configuration" %}
+
+{% raw %}
+
+```yaml
+alias: "Notify when program ends"
+triggers:
+  - trigger: state
+    entity_id:
+      - sensor.washing_machine
+    to: program_ended
+actions:
+  - service: notify.notify
+    data:
+      message: "The appliance has finished the program."
+```
+
+{% endraw %}
+{% enddetails %}
+
+## Data updates
+
+This integration uses server-sent events from the Miele API to receive live updates from the appliances.
+When the configuration entry is loaded or after a streaming error (for example after disconnection), the integration will request all data (such as appliance info, available commands, programs, settings, and status) for all appliances.
+
+## Known limitations
+
+- The Miele 3rd party API does not fully match the Miele app. Some programs, options, or settings available in the app may not be accessible or usable via the API.
+- This integration supports only one integration entry, as the Miele 3rd party API does not allow for the unique identification of an account.
+
+## Troubleshooting
+
+### Unavailable entities for a device
+
+#### Symptom: "The entities related to an appliance were available but no longer are"
+
+After reloading the Miele integration, the entities related to an appliance that used to be available are no longer available.
+
+##### Description
+
+Unavailable entities can have multiple causes:
+
+- The appliance is turned off. When it is turned off, the appliance is disconnected and the API does not retrieve information about the appliance.
+- The appliance is experiencing a network issue.
+- The Miele API is experiencing issues.
+
+##### Resolution
+
+To try to solve the above issues, follow these steps:
+
+1. Turn on the appliance and reload the Miele integration.
+2. If the appliance is turned on and the issue persists, check the network connection of the appliance and perform a soft reset on the appliance.
+3. If the issue persists, check the connection of the appliance with the Miele API by checking it in the Miele app.
+   1. Open the Miele app.
+   2. Go to the appliance that is experiencing the issue.
+   3. Press the cog-wheel to view more information.
+4. If everything is correct and the issue persists, contact Miele support.
+   - [Miele service and contact](https://www.miele.com/)
+   - [Miele developer Help & Support](https://www.miele.com/developer)
+
+## Removing the integration
+
+This integration follows standard integration removal. No extra steps are required.
+
+{% include integrations/remove_device_service.md %}
```

Comment:
I would also mention that if you use your own application credentials, that they can be removed at devices & Services -> 3 dots -> Application credentials.

I have so many people who create mistakes with that

Reply:
Many, many users of the cc also.